dataland-frontend/tests/e2e/specs/high/Prepopulation.ts

```diff
@@ -12,56 +19,91 @@ describe('Population Test', () => {
 
     });
 
+    async function uploadData(dataArray:Array<object>, endpoint:string){
```

Comment:
Warum ist die Function async? Die awaitet doch brav dass alles hochgeladen ist.

Reply:
nur innerhalb einer async Funktion kann man `await` implementieren, sonst gibt es Fehler und Kompiler wird meckern. In Zeile 27 und 28 bräuchte man await für synchronization

Reply:
ok